karma.conf.js

```diff
@@ -30,7 +43,27 @@ module.exports = function (karmaConfig) {
 
     // Test results reporter to use
     // Possible values: 'dots', 'progress', 'junit', 'growl', 'coverage'
-    reporters: ['progress'],
+    reporters: karmaReporters,
+
+    // Configure the coverage reporter
+    coverageReporter: {
+      dir : 'coverage/',
+      reporters: [
```

Comment:
we can add or remove options here. I think we can get away with html and text only. Text will output to the console for us. Does anyone want lcov?